src/modules/chat_scrollback_settings/index.js

```diff
@@ -0,0 +1,47 @@
+const watcher = require('../../watcher');
+const storage = require('../../storage');
+const twitch = require('../../utils/twitch');
+
+const DEFAULT_SIZE = 200;
+const PROMPT_BODY = `Change the number of messages the chat is able to scrollback to. Increasing this value can slow down your browser...
+Leaving this value empty restore default settings.
+`;
+
+function changeSizeSettings(promptBody, storageID) {
+    const size = prompt(promptBody, storage.get(storageID) || DEFAULT_SIZE);
+    if (size !== null) {
+        storage.set(storageID, size.trim());
+    }
+}
+
+function updateSizeSettings() {
+    const sizeString = storage.get('chatScrollbackSize') || '';
+    let size;
+    if (sizeString.trim().length === 0) {
+        size = DEFAULT_SIZE;
+    } else {
+        try {
```

Comment:
i dont get why the business logic of parsing is here. it should be coerced after the prompt to change, then stored as a number